Makefile

```diff
@@ -10,7 +10,7 @@ vramalloc_OBJS = vramalloc.o
 VRAM_OBJS   = vram.o
 
 INCDIR = 
-CFLAGS = -G0 -mno-explicit-relocs -O2 -Wall -Werror -g
+CFLAGS = -mno-gpopt -mno-explicit-relocs -O2 -Wall -Werror -g
```

Comment:
could you remove the `-mno-explicit-relocs` and then do the same to CMakeLists.txt

Reply:
gonna do the cmake one in a separate pr since i used github edit frontend for this one, otherwise i'd have to switch to the git cli